mithril-test-lab/mithril-end-to-end/backward-compatibility.md

```diff
@@ -0,0 +1,68 @@
+# Backward compatibility
+
+The end-to-end tests scenarios are backward compatible with previous nodes versions.
+
+This is achieved by differentiating the scenarios based on the version of the nodes they are testing.
+
+There are multiple ways to do this:
+
+- disabling part of the scenarios
+- adding or removing arguments to the binaries (i.e. `--backend` flag for the client-cli)
+
+> [!TIP]
+> Node versions and distribution versions below are the ones that first included the changes, has node versions evolve
+> faster than distribution versions the actual node version included in the distribution may be higher.
+>
+> i.e. `--origin-tag` flag for the client-cli was introduced in version `0.11.13`, the first distribution that included
+> that change is `2517.0`, which include client-cli `0.12.0`.
+
+---
+
+## Supported changes
+
+List of breaking changes that are supported by the end-to-end tests.
+
+format is: `- since 'X.Y.Z' (distribution version) [to 'X.Y.Z' (distribution version) (optional)]: supported change`
+
+### Mithril client
+
+- since `0.12.34` (2543.0): test of new `--epoch` filter to `cardano-db list` (disabled on lower versions)
```

Comment:
This will make the lists a bit more readable:
```suggestion
- **since `0.12.34` (2543.0)**: test of new `--epoch` filter to `cardano-db list` (disabled on lower versions)
```